main.cpp

```diff
@@ -52,84 +52,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: amp
+//  action 1: distorts sound
+amp.distort();
+//  action 2: mutes sound
+amp.mute();
+//  action 3: adds chorus
+ amp.addChorus();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: cloud
+//  action 1: creates rain
+cloud.rain();
+
+//  action 2: creates snow
+cloud.snow();
+//  action 3: evaporates
+cloud.evaporate(); 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: disc golfer
+//  action 1: throws disc
+discGolfer.throw();
+//  action 2: runs
+discGolfer.run();
+//  action 3: putts
+ discGolfer.putt();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: guitar player
+//  action 1: plays guitar
+guitarPlayer.playGuitar();
+//  action 2: shreds
+guitarPlayer.shred();
+//  action 3: loses pick
+ guitarPlayer.losePick();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: boss
+//  action 1: hires person
+boss.hirePerson();
+//  action 2: fires person
+boss.firePerson();
+//  action 3: quits job
+ boss.quitJob();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: bird
+//  action 1: caws
+bird.caws();
+//  action 2: chirps
+bird.chirps();
+//  action 3: fly
+ bird.fly();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: activist
+//  action 1: rallies
+activist.rally();
+//  action 2: blocks traffic
+activist.blockTraffic();
+//  action 3: start revolution
+ activist.startRevolution();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.